### DISCUSSION TITLE
Is there any CMS/Markdown editor inside? Or recommended ouside?

### DISCUSSION THREAD

**Discussion by alibabagogogo** [alshedivat/al-folio#1038] [Q&A] [answered]
Is there any CMS/Markdown editor inside? Or recommended?

**Comment by george-gca** [comment, marked as answer]
The recommended way to properly edit and visualize how your site will look like is to run in locally. I did a [small tutorial](https://george-gca.github.io/blog/2022/running-local-al-folio/) on how to achieve this, but the current recommended way is to use [Docker](https://github.com/alshedivat/al-folio#local-setup-using-docker-recommended-on-windows) on all platforms, as per #1523.
